spec/requests/api/v1/items_request_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe "Items API", type: :request do 
+  before(:each) do
+    @merchant = Merchant.create(name: "Awesome Merchant") 
+    @item1 = Item.create(
+      name: "Catnip Toy",
+      description: "A soft toy filled with catnip.",
+      unit_price: 12.99,
+      merchant_id: @merchant.id
+    )
+
+    @item2 = Item.create(
+      name: "Laser Pointer",
+      description: "A laser pointer to keep your cat active.",
+      unit_price: 9.99,
+      merchant_id: @merchant.id
+    )
+
+    @item3 = Item.create(
+      name: "Feather Wand",
+      description: "A wand with feathers to entice your kitty.",
+      unit_price: 15.50,
+      merchant_id: @merchant.id
+    )
+  end
+
+  it 'can fetch all items' do
```

Comment:
Since this is fetching all, we need to validate that it has all of the records and that they are all correct, check out the  "can fetch multiple merchants" test in merchants_request_spec

Reply:
Ill check that out, thank you.

Reply:
You may have to much logic in the index.  Can the logic move to the model, and the index just calls a sorted method?